sdk/keyvault/azure-keyvault-secrets/README.md

```diff
@@ -17,6 +17,7 @@ create, manage, and deploy public and private SSL/TLS certificates
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```

Comment:
We can also update the version mentioned in the `Prerequisites` section, like you did for `azure-keyvault-certificates` and `azure-keyvault-keys`

Reply:
Thank you so much for the catch! I have updated the version in the `Prerequisites`!